tests/compiletest.rs

```diff
@@ -37,6 +37,9 @@ fn run_tests(mode: &str, path: &str, target: &str, mut flags: Vec<String>) {
         flags.push(format!("--sysroot {}", sysroot));
     }
 
+    // Add a test env var to do environment communication tests
+    std::env::set_var("MIRI_ENV_VAR_TEST", "0");
```

Comment:
Could you set this globally, in `test_runner`? After all, this does affect everything that runs.

Reply:
What do you mean with this? I'm not that familiar with the testing part of miri

Reply:
I mean just move these two lines to `fn test_runner`.